### PR TITLE
p;arsing error fix from mysticpants develop branch

### PR DESCRIPTION
**Pull request by betzrhodes**

### CODE REVIEW

**Comment by betzrhodes** [comment]
This fix seems to be on mystic pants branch but not ours.  Changes look like they fix the issue filed when we reviewed the library, but never got bubbled up to our repo.

**Comment by betzrhodes** [comment]
when library update is available we should update this forum post.
https://forums.electricimp.com/t/modbus-tcp/4401

File: Modbus485Master/README.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ modbus.maskWriteRegister(0x01, 0x10, 0xFFFF, 0x0000, function(error, result) {
 
 Function Code : 23
 
-This method performs a combination of one read operation and one write operation in a single Modbus transaction. The write operation is performed before the read. It takes the following parameters:
+This method performs a combination of one read operation and one write operation in a single Modbus transaction. The write operation is performed before the read ^. It takes the following parameters:
```
Comment (ppetrosh) [inline Modbus485Master/README.md:256]: Is this changed by mistake?

File: ModbusTCPMaster/README.md [hunk 2]
```diff
@@ -286,7 +289,7 @@ modbus.maskWriteRegister(0x10, 0xFFFF, 0x0000, function(error, result) {
 
 Function Code : 23
 
-This method performs a combination of one read operation and one write operation in a single Modbus transaction. The write operation is performed before the read. It takes the following parameters:
+This method performs a combination of one read operation and one write operation in a single Modbus transaction. The write operation is performed before the read ^. It takes the following parameters:
```
Comment (ppetrosh) [inline ModbusTCPMaster/README.md:292]: Typo?

**Review by ppetrosh — COMMENTED** [review]
Looks good to me except for the two minor comments.

**Pull request merged**